Formula/curlcpp.rb

```diff
@@ -0,0 +1,25 @@
+class Curlcpp < Formula
+  desc "Object oriented C++ wrapper for CURL (libcurl)"
+  homepage "https://josephp91.github.io/curlcpp"
+  url "https://github.com/JosephP91/curlcpp/archive/refs/tags/1.5.tar.gz"
+  sha256 "d1c236be862965d5882a842be3112b16e7f49d6fdff9906f7aec71ee8669d60e"
+  license "MIT"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "curl"
+
+  def install
+    ENV.deparallelize
+    Dir.mkdir("build")
+    chdir "build" do
+      system "cmake", "..", *std_cmake_args
+      system "make"
+      system "make", "install"
+    end
```

Comment:
Please use https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project and https://cmake.org/cmake/help/latest/manual/cmake.1.html#install-a-project instead